cmd/version_test.go

```diff
@@ -0,0 +1,18 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/mod/semver"
+	"testing"
+)
+
+func TestVersion(t *testing.T) {
+
+	t.Run("Check Version is set.", func(t *testing.T) {
+		fmt.Println(fmt.Sprintf("Version is %s", Version))
```

Comment:
Minor: linter suggests: `fmt.Printf("Version is %s\n", version)`

Reply:
Not a fan of "\n" in the text, prefer Println to take care of that.